orttraining/orttraining/python/training/_ortmodule_training_manager.py

```diff
@@ -183,8 +186,28 @@ def _create_execution_agent(self):
         """Creates a TrainingAgent that can run the forward and backward graph on the training model"""
 
         session_options, providers, provider_options = self._get_session_config()
+
+        fw_feed_names = []
+        for input in self._optimized_onnx_model.graph.input:
+            fw_feed_names.append(input.name)
```

Comment:
could be more pythonic:
fw_feed_names = [input.name for input in self._optimized_onnx.model.graph.input]